django_project/core/custom_middleware.py

```diff
@@ -163,5 +163,5 @@ def process_request(self, request):
             else:
                 # for production the domain is hardcoded for consistency
                 return HttpResponseRedirect(
-                    'http://changelog.qgis.org/en/domain-not-found/'
+                    'http://changelog.kartoza.com/en/domain-not-found/'
```

Comment:
Could you do a follow up commit that takes this out of settings rather? I don't think it is good to hard code URLs into the code.